googleapis/google-cloud-node
# migrate code from googleapis/nodejs-dataflow

- feat: initial stub of library
- feat: push templated files
- feat: add initial samples and tests (#2)
- chore(deps): update dependency sinon to v11 (#5)
- chore: release 1.0.0 (#1)
- fix(deps): google-gax v2.17.1 (#6)
- fix: Updating WORKSPACE files to use the newest version of the Typescript generator. (#8)
- chore: release 1.0.1 (#7)
- chore(nodejs): update client ref docs link in metadata (#13)
- fix(build): migrate to using main branch (#15)
- fix(deps): google-gax v2.24.1 (#17)
- chore: release 1.0.2 (#16)
- feat: turns on self-signed JWT feature flag (#18)
- chore: release 1.1.0 (#20)
- build(node): update deps used during postprocessing (#1243) (#28)
- chore: fix the wrong post processor image (#30)
- chore(deps): update dependency @types/node to v16 (#29)
- chore(deps): update dependency sinon to v12 (#31)
- docs(samples): add example tags to generated samples (#32)
- build: add generated samples to .eslintignore (#33)
- docs(node): support "stable"/"preview" release level (#1312) (#36)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#37)
- chore: add api_shortname and library_type to repo metadata (#35)
- test(nodejs): remove 15 add 16 (#1322) (#39)
- chore: update v2.12.0 gapic-generator-typescript (#43)
- feat: new parameters in FlexTemplateRuntimeEnvironment
- docs(nodejs): version support policy edits (#1346) (#47)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#48)
- chore: update v2.14.1 gapic-generator-typescript (#49)
- feat: Add the ability to plumb environment capabilities through v1beta3 protos. (#53)
- chore(main): release 1.2.0 (#45)
- chore(deps): update dependency sinon to v13 (#52)
- build(node): update client library version in samples metadata (#1356) (#60)
- chore(deps): update dependency @types/mocha to v9 (#61)
- chore(deps): update dependency sinon to v14 (#64)
- build!: update library to use Node 12 (#65)
- fix: fixes for dynamic routing and streaming descriptors (#68)
- chore(deps): update dependency jsdoc-fresh to v2 (#70)
- chore(deps): update dependency jsdoc-region-tag to v2 (#71)
- chore(main): release 2.0.0 (#67)
- fix(docs): document fallback rest option (#72)
- chore(deps): update dependency linkinator to v4 (#75)
- fix: better support for fallback mode (#76)
- fix: change import long to require (#77)
- fix: do not import the whole google-gax from proto JS (#1553) (#79)
- fix: allow passing gax instance to client constructor (#80)
- fix: preserve default values in x-goog-request-params header (#81)
- test: use fully qualified request type name in tests (#83)
- fix(deps): use google-gax v3.5.2 (#87)
- chore(deps): update dependency @types/node to v18 (#85)
- chore(deps): update dependency jsdoc to v4 (#89)
- fix: regenerated protos JS and TS definitions (#90)
- chore(main): release 2.0.1 (#73)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
